linalg/solvers.cpp

```diff
@@ -2149,6 +2161,8 @@ void LBFGSSolver::Mult(const Vector &b, Vector &x) const
    {
       mfem::out << "LBFGS: No convergence!\n";
    }
+
+   Monitor(it, norm, r, x, true);
```

Comment:
LBFGS does not have monitoring, either add monitor to the main loop or remove this.